tools/c7n_azure/c7n_azure/autoscale_utils.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from c7n.utils import local_session
+from c7n_azure.session import Session
+from azure.mgmt.monitor import MonitorManagementClient
+
+
+class AutoScaleUtilities(object):
+    log = logging.getLogger('custodian.azure.autoscale_utils')
```

Comment:
@logachev @edonkor1 does this follow naming convention?

Reply:
I'm fine with using underscore